packages/components/src/atoms/atoms.css.ts

```diff
@@ -123,24 +94,20 @@ const unresponsiveStyles = createAtomicStyles({
 
 const responsiveStyles = createAtomicStyles({
   conditions: {
-    sm: {},
-    md: {'@media': `screen and (min-width: ${screens['screen-md']})`},
-    lg: {'@media': `screen and (min-width: ${screens['screen-lg']})`},
-    xl: {'@media': `screen and (min-width: ${screens['screen-xl']})`},
-    '2xl': {'@media': `screen and (min-width: ${screens['screen-2xl']})`},
+    xs: {},
```

Comment:
```suggestion
    xs: {'@media': `screen and (min-width: ${breakpoints.xs})`},
```
This is effectively the same thing, but might be worth adding for consistency.

Reply:
Was curious about this one so I did a little try, build, and explore on the changes.

Looking at the compiled CSS, adding the `@media': screen and (min-width: ${screens['screen-lg']})`  wraps all the atoms in the default media query. Which does end in the same end effect, but I partial to keeping as-is to avoid the extra media query wrapping.

Reply:
Does the `xs` property only exist then to be a `defaultCondition`? Would it work the same way if we removed both of those? Just wondering if this adds to confusion if `xs` can never actually be used

Reply:
`xs` was influenced by [material-ui's breakpoint scale](https://material-ui.com/customization/breakpoints/#breakpoints) which takes a mobile first approach. Although, `xs` may seem extraneous it can be helpful for breakpoint utilities (e.g. `theme.breakpoin.down('xs')` => `@media (max-width: 599px)`)